default/python/common/configure_logging.py

```diff
@@ -1,26 +1,124 @@
+"""
+configure_logging redirects print and the logger to use the freeorion server.
```

Comment:
> Do we use different log level for release builds?

Not yet and good part of the C++ internal logs to debug when another log level would be more appropriate, but I claim that it shouldn't stay this way.